locale/action_names.rb

```diff
@@ -1,61 +1,73 @@
 # Autogenerated!
-_('Delete Lifecycle Environment')
-_('Destroy Content Host')
-_('Hypervisors')
-_('Hypervisors update')
-_('Install package')
-_('Remove package')
-_('Update for host')
-_('Install Applicable Errata')
-_('Update package')
-_('Update')
-_('Install package group')
-_('Package Profile Update')
-_('Configure capsule')
-_('Auto-attach subscriptions')
-_('Create repos')
-_('Create')
-_('Remove package group')
-_('Install erratum')
-_('Register Host')
-_('Remove subscriptions')
-_('Unregister Host')
-_('Delete Activation Key')
-_('Abstract async task')
-_('Incremental Update')
-_('Destroy')
-_('Synchronize capsule content')
-_('Product Create')
-_('Export')
-_('Delete Product')
-_('Delete')
-_('Auto attach subscriptions')
-_('Reindex subscriptions')
-_('Errata mail')
-_('Attach subscriptions')
-_('Incremental Update of 0 Content View Version(s) ')
-_('Import Puppet classes')
-_('Import facts')
-_('Filtered index content')
-_('Upload into')
-_('Action with sub plans')
-_('Upload errata into')
-_('Remove from Environment')
-_('Sync SUSE subscriptions')
-_('Enable')
-_('Index errata')
-_('Remove Version')
-_('Index package groups')
-_('Add Sync Plan Products')
-_('Sync SUSE subscriptions (Products)')
-_('Remove Content')
-_('Index content')
-_('Disable')
-_('Sync SUSE subscriptions (Repositories)')
-_('Subscribe SCC Product')
-_('Remove Versions and Associations')
-_('Synchronize')
-_('Publish')
-_('Update Sync Plan Products')
-_('Promotion')
-_('Update Sync Plan')
+_("Incremental Update")
```

Comment:
I am commenting on this line as an example.
There appear to be a lot of strings in this automatically generated file, that don't appear anywhere in the plugin. (I think they are from Katello).

`"Incremental Update"` is just one example.
By comparing with strings found using `git grep '_('` it looks like about half these strings should not be here.

Reply:
Solved by manually removing the offending strings.